build/pkgs/openblas/spkg-install.in

```diff
@@ -34,10 +34,18 @@ fi
 
 echo "Building OpenBLAS: $MAKE $OPENBLAS_CONFIGURE"
 
+if $MAKE --version | grep -q -F '3.81'; then
+    # Work around https://savannah.gnu.org/bugs/?15919
+    OPENBLAS_CONFIGURE+=" MAKE_NB_JOBS=1"
+else
+    # Do not emit "-j" options; use jobserver
+    OPENBLAS_CONFIGURE+=" MAKE_NB_JOBS=0"
+fi
+
 # Ensure USE_TLS=1 ; see https://github.com/sagemath/sage/issues/27256
 OPENBLAS_CONFIGURE="$OPENBLAS_CONFIGURE USE_TLS=1"
 
-if ! (sdh_make libs netlib shared $OPENBLAS_CONFIGURE); then
+if ! (sdh_make libs $OPENBLAS_CONFIGURE && sdh_make netlib $OPENBLAS_CONFIGURE && sdh_make shared $OPENBLAS_CONFIGURE); then
```

Comment:
Why do we need to separate the `make` targets like this?

Reply:
This change appears to fix the build failures mentioned in https://github.com/sagemath/sage/pull/36671#issuecomment-1799307062